docker/android/tests/Android.mk

```diff
@@ -0,0 +1,101 @@
+LOCAL_PATH := $(call my-dir)
```

Comment:
Likewise, we want to check our Makefile "parsers" work, especially if we have conditional directives that could be removed by a comment. We don't want to generate unbalanced directives, so we have testing to ensure this.